quickstart/monitoring/monitoring-quickstart.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>Is record-level custom metrics available to include here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
No, this is purposefully removed from QuickStart docs as we wanted to keep this simple.